pkg/subscriptions/subscriptions-register.jsx

```diff
@@ -0,0 +1,239 @@
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                ];
+      /*
+      <tr>
+                        <td />
+                        <td className="top form-td-ct-subscriptions">
+                            <label className="control-label form-tr-ct-title" for="subscription-proxy-server">
+                                {_("Server")}
+                            </label>
+                        </td>
+                        <td>
+                            <input id="subscription-proxy-server" className="form-control" type="text"
+                                   placeholder="hostname:port" value={this.props.proxyServer}
+                                   onChange={this.props.onChange.bind(this, 'proxyServer')}/>
+                        </td>
+                    </tr>,
+                    <tr>
+                        <td />
+                        <td className="top form-td-ct-subscriptions">
+                            <label className="control-label form-tr-ct-title" for="subscription-proxy-user">
+                                {_("User")}
+                            </label>
+                        </td>
+                        <td>
+                            <input id="subscription-proxy-user" className="form-control" type="text"
+                                   value={this.props.proxyUser}
+                                   onChange={this.props.onChange.bind(this, 'proxyUser')}/>
+                        </td>
+                    </tr>,
+                    <tr>
+                        <td />
+                        <td className="top form-td-ct-subscriptions">
+                            <label className="control-label form-tr-ct-title" for="subscription-proxy-password">
+                                {_("Password")}
+                            </label>
+                        </td>
+                        <td>
+                            <input id="subscription-proxy-password" className="form-control" type="password"
+                                   value={this.props.proxyPassword}
+                                   onChange={this.props.onChange.bind(this, 'proxyPassword')}/>
+                        </td>
+                    </tr>
+      */
+            }
+            return (
+                <div className="modal-body">
+                    <table className="form-table-ct">
+                        <tr>
+                            <td className="top">
+                                <label className="control-label" for="subscription-register-url">
+                                    {_("URL")}
+                                </label>
+                            </td>
+                            <td>
+                                <Select.Select key='urlSource' onChange={ this.props.onChange.bind(this, 'url') }
```

Comment:
The select box in the view shows "default" (lower case) originally. Once I select "Custom URL", and go back, then it's capitalized properly. Should be capitalized properly when opening the dialog the first time.